extensions/amp-story-player/validator-amp-story-player.protoascii

```diff
@@ -42,4 +42,49 @@ descendant_tag_list {
   tag: "A"
   tag: "SPAN"
   tag: "I-AMPHTML-SIZER"  # Only allowed when document is transformed AMP
+  tag: "IMG"
+}
+tags: {
+  html_format: AMP
+  tag_name: "IMG"
+  mandatory_ancestor: "AMP-STORY-PLAYER"
+  attrs: {
+    name: "loading"
+    value: "lazy"
+  }
+  attrs: { name: "alt" }
+  attrs: { name: "attribution" }
+  attrs: { name: "border" }  # Not valid html5 but commonly used and supported.
+  attrs: {
+    name: "decoding"
+    value: "async"
+    value: "auto"
+    value: "sync"
+  }
+  attrs: { name: "height" }  # Not valid html5 but commonly used and supported.
+  attrs: { name: "ismap" }
+  attrs: {
+    name: "longdesc"
+    value_url: {
+      protocol: "http"
+      protocol: "https"
+    }
+    disallowed_value_regex: "__amp_source_origin"
+  }
+  attrs: { name: "importance" }  # Not yet part of the html spec
+  attrs: { name: "intrinsicsize" }  # Not yet part of the html spec
+  attrs: {
+    name: "src"
+    alternative_names: "srcset"
+    mandatory: true
+    value_url: {
+      protocol: "data"
+      protocol: "http" # For parity with amp-img's mandatory-src-or-srcset.
+      protocol: "https"
+      allow_relative: true
+    }
+    disallowed_value_regex: "__amp_source_origin"
+  }
+  attrs: { name: "sizes" }
+  attrs: { name: "width" }  # Not valid html5 but commonly used and supported.
```

Comment:
Q for @ampproject/wg-caching people, do we have to repeat all these rules?

Reply:
If there are shared attributes between tagspecs then consider using `attr_lists`. See https://github.com/ampproject/amphtml/blob/master/validator/validator.proto#L200

For example, the `src` attribute here mirrors exactly the existing `mandatory-src-or-srcset` attribute list that is in [validator-main.protoascii](https://github.com/ampproject/amphtml/blob/master/validator/validator-main.protoascii#L4138). Note `allow_relative` is default `true`.